Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12.16.3
+LABEL maintainer=Netlify
+
+RUN apt update && apt install -y jq
+
+COPY plugins.json .
+RUN jq -r 'map(.package + "@" + .version) | join(" ")' plugins.json | xargs npm install
```

Comment:
We want to make sure is that all plugins are installed using the same Node.js version as `@netlify/build` and `@netlify/config`, i.e. `12.6.3` at the moment.

The previous code was using nvm for this, whereas this is using the Docker base image `node:12.16.3`. I think this should work (and is in fact much simpler), but we might want to make some additional testing about this specific requirement.

Reply:
Tested, and this works :+1: